### ISSUE TITLE
make a local-only version

### ISSUE THREAD

**Issue by ykhli** [a16z-infra/companion-app#58]

**Comment by shivvamm** [comment]
can you tell me more about it like locally means what like a pwa version of it

**Comment by shivvamm** [comment]
i want to contribute on this issue

**Comment by KevinColemanInc** [comment]
I think they mean to drop all the 3rd party signups and use local tools (local redis, postgres, etc.) instead of upstash, supabase, etc.

**Comment by rxwf001** [comment]
Exactly. Actually, I'm finding a way to use a local AI too. This project is very good because they use a conversational memory logic
